hotsos/defs/scenarios/openstack/nova/bugs/lp2012284.yaml

```diff
@@ -0,0 +1,18 @@
+checks:
+  has_2012284:
+    input:
+      path: 'var/log/nova/nova-compute.log'
+    expr: ".+Failure prepping block device: gi.repository.GLib.GError: g-io-error-quark: Error opening directory '/usr/share/osinfo': Permission denied"
```

Comment:
I think this should have a timestamp match, at least all the existing nova ones have something like this:

([\d-]+) ([\d:]+)\.\d{3} .+Failure...

Reply:
done. However, I had to change the quotation marks and the ' in the message to a . because they cannot be escaped in yaml (as far as I could find)